### DISCUSSION TITLE
transferring to new server

### DISCUSSION THREAD

**Discussion by PaulMcF1987** [Budibase/budibase#6431] [Help]
I am in the process of moving from a VPS to a much larger dedicated server.
Is there a recommended way to move budibase and all created apps over to the new server without losing any information?

**Comment by melohagan** [comment]
A full backup of the volume might be prudent: 

- https://docs.docker.com/desktop/backup-and-restore/
- https://docs.docker.com/storage/volumes/#backup-restore-or-migrate-data-volumes

**Reply by PaulMcF1987** [reply]
@melohagan 
Would [this](https://www.youtube.com/watch?v=QskmB4fb-uo) work?
I was going to ty this, but didnt want to incase it meant that I was unable to update the images to newer versions.
